var/spack/repos/builtin/packages/git/package.py

```diff
@@ -56,7 +55,7 @@ class Git(Package):
     # Also depends_on gettext: apt-get install gettext (Ubuntu)
 
     # Use system perl for now.
-    # depends_on("perl")
+    depends_on("perl")
     # depends_on("pcre")
```

Comment:
There is actually a pcre package now. Can you use that too and make sure it works?